ecosystem/attestation-station/using-sdk/README.md

```diff
@@ -0,0 +1,234 @@
+# AttestationStation SDK
+
+[![Discord](https://img.shields.io/discord/667044843901681675.svg?color=768AD4&label=discord&logo=https%3A%2F%2Fdiscordapp.com%2Fassets%2F8c9701b98ad4372b58f13fd9f65f966e.svg)](https://discord-gateway.optimism.io)
+[![Twitter Follow](https://img.shields.io/twitter/follow/optimismFND.svg?label=optimismFND&style=social)](https://twitter.com/optimismFND)
+
+The [AttestationStation smart contract](https://github.com/ethereum-optimism/optimism/blob/develop/packages/contracts-periphery/contracts/universal/op-nft/AttestationStation.sol) contains a public `attestations` mapping that anyone can write to and read from. 
+For more context on the AttestationStation visit the [overview in our developer documentation](https://community.optimism.io/docs/governance/attestation-station/).
+In this tutorial you learn how to read, interpret, and write those attestations.
+
+The contract we'll be using is on the Optimism Goerli network, at address [`0xEE36eaaD94d1Cc1d0eccaDb55C38bFfB6Be06C77`](https://goerli-explorer.optimism.io/address/0xEE36eaaD94d1Cc1d0eccaDb55C38bFfB6Be06C77).
+On the production Optimism network the contract is [at the same address](https://explorer.optimism.io/address/0xEE36eaaD94d1Cc1d0eccaDb55C38bFfB6Be06C77)
+
+This is one of the methods to access AttestationStation (ATST).
+[There are other ways](../README.md).
+
+## Prerequisites
+
+- You have [Node.js](https://nodejs.org/en/) running on your computer, as well as [yarn](https://classic.yarnpkg.com/lang/en/).
+- There is network connectivity to a provider on the Optimism Goerli network, and to the npm package registry.
+
+
+## Running the app
+
+1. Use `yarn` to download the packages you need
```

Comment:
Would recomend doing tutorials in future with npm.  Out of these 4 choices: npm, pnpm, bun, and yarn, `yarn` is the worst choice for package manager.   Also you are using yarn classic which is no longer maintained

Reply:
Why is `npm` superior? I thought `yarn` was multi-threaded.